src/dotnetsheff.Api/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Why do we have another .gitignore file?

Reply:
we don't need it. VS added it when i created an new Azure Functions v2 functions. I'll get rid of it